### PR TITLE
Have a 404 page when site is not reachable #37

### LINKED ISSUE #37: Have a 404 page when site is not reachable
* Currently no 404 page is available in case site is down or unreachable 
* A similar error routing has been done in PR #35 in case of db fault 
- [ ] Create a 404 error page for whenever site is unreachable

### PR DESCRIPTION
**Pull request by anand-nakat**
Fixes #37 
1. The site redirects to 404 Page if the url typed by user doesn't exist. 

image

### CODE REVIEW

File: 404.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            color: black;
+            font-size: 2.4rem;
+
+        }
+    </style>
+    <script>
+        setTimeout(function(){
+            window.location.href='login.php';
+        },5000);
+    </script>
```
Comment (anand-nakat) [inline 404.html:63]: After 5 sec, the page will automatically redirect back to login page

File: .htaccess [hunk 2]
```diff
@@ -0,0 +1 @@
+ErrorDocument 404 /404.html
```
Comment (anand-nakat) [inline .htaccess:1]: .htaccess code for redirection to 404 page

File: db-error.html [hunk 3]
```diff
@@ -27,28 +27,40 @@
 		<link rel="stylesheet" type="text/css" href="css/util.css">
         <link rel="stylesheet" type="text/css" href="css/register.css">
     <style>
+        *{
+            font-family: 'Poppins';
+        }
+
         body{
-            background: #474787 radial-gradient(circle at 60% 100%, #474787 5%, #2c2c54 94%);
+            background:radial-gradient(circle at 60% 100%,#5245b9  5%, #4b458e 94%);
```
Comment (anand-nakat) [inline db-error.html:35]: Changed Background for db-error page.. Now similar to 404 page

File: 404.html (hunk 1 above)
Comment (Chintan2108) [reply 404.html:63]: Good Addendum!

**Review by Chintan2108 — APPROVED** [review]
PR credit subject to successful run on GCP

**Comment by Chintan2108** [comment]
give @anand-nakat 25 points

**Pull request merged**